### PR TITLE
[SECRET-31] Add Secret Store documentation to dev-center

### PR DESCRIPTION
**Pull request by ZachEddy**
This PR adds documentation to the `dev-center` for Secret Store-related functionality. Secret Store is a new feature that lets users store passwords, credentials, etc. on Algorithmia so they don't have to hardcode sensitive data in their source code.

### CODE REVIEW

File: _pages/platform/algorithm-secrets.md [hunk 1]
```diff
@@ -0,0 +1,51 @@
...
+Start by clicking "New secret" to create an algorithm secret. Next, give your secret a name, description, key, and value. The key corresponds to the environment variable name and the value is the secret itself.
+
+It can take up to one minute for new secrets to show up inside your algorithm. Similarly, when you update a secret, it can take one minute for that change to take effect.
+{: .notice-info}
+
+![Image of UI create secret page](/developers/images/post_images/algorithm_secrets/create_secret_page.png)
+
+After your secret is configured, it can be updated or deleted on the settings page. Clicking "Update" will direct you to a form that is identical to the form shown above, and you'll be able to change the secret's name, description, and value.
+
+![Image of UI settings page for creating secrets](/developers/images/post_images/algorithm_secrets/settings_page_with_secret.png)
```
Comment (bnlb) [inline _pages/platform/algorithm-secrets.md]: This alt tag is identical to a previous one. I'd update to something that reflects that a secret exists on the page now.
```suggestion
![Image of UI settings page with an existing secret](/developers/images/post_images/algorithm_secrets/settings_page_with_secret.png)
```

File: _pages/platform/algorithm-secrets.md [hunk 2]
```diff
@@ -0,0 +1,51 @@
...
+
+## Managing algorithm secrets
+
+Secrets are managed on the algorithm settings page. Here you'll be able to create, update, delete, and view your existing algorithm secrets. The steps below demonstrate the process for creating secrets.
+
+![Image of UI settings page for creating secrets](/developers/images/post_images/algorithm_secrets/settings_page.png)
+
+Start by clicking "New secret" to create an algorithm secret. Next, give your secret a name, description, key, and value. The key corresponds to the environment variable name and the value is the secret itself.
+
+It can take up to one minute for new secrets to show up inside your algorithm. Similarly, when you update a secret, it can take one minute for that change to take effect.
```
Comment (bnlb) [inline _pages/platform/algorithm-secrets.md:44]: Smart addition

**Review by bnlb — DISMISSED** [review]
🔐

**Comment by ZachEddy** [comment]
Thanks for the quick review, @bnlb! I updated the `alt-tag` like you mentioned. I met with @algorithmiaio/product on Friday and they requested I callout enterprise support more explicitly, so I added the `External secret stores` heading.

**Review by stevielkim — DISMISSED** [review]
Awesome job on the docs!

**Comment by ZachEddy** [comment]
Thank you, @StephanieKim! I was just about to merge this pull request, but I remembered there were some recent secret store UI changes. I updated one of the `.png` images in 603a6ab to reflect the new UI.

I dismissed your review by pushing changes, so would you mind re-approving?

**Review by bnlb — APPROVED** [review]

**Pull request merged**